truss/templates/control/control/helpers/truss_patch/model_container_patch_applier.py

```diff
@@ -29,11 +35,14 @@ def __init__(
         if pip_path is not None:
             self._pip_path_cached = "pip"
 
-    def apply_patch(self, patch: Patch):
+    def __call__(self, patch: Patch):
         self._app_logger.debug(f"Applying patch {patch.to_dict()}")
         if isinstance(patch.body, ModelCodePatch):
             model_code_patch: ModelCodePatch = patch.body
             self._apply_model_code_patch(model_code_patch)
```

Comment:
Remove unnecessary `self._apply_model_code_patch(model_code_patch)` and definition on line 56 of this file in favor of the import

Reply:
good catch, I forgot to remove this.